osgar/test_bus.py

```diff
@@ -3,9 +3,9 @@
 from queue import Queue
 from datetime import timedelta
 
-from osgar.drivers.bus import (BusHandler, BusShutdownException,
-                               LogBusHandler, LogBusHandlerInputsOnly,
-                               serialize)
+from osgar.bus import (BusHandler, BusShutdownException,
+                       LogBusHandler, LogBusHandlerInputsOnly,
+                       serialize)
```

Comment:
You seem to be liking the naked imports a lot 😄 . Something to think about:

```python
from osgar import bus
```
further in code use
```python
bus.Handler
bus.ShutDownException
bus.LogHandler
bus.LogHandlerInputsOnly
```